examples/03_connectivity/plot_data_driven_parcellations.py

```diff
@@ -2,30 +2,31 @@
 Clustering methods to learn a brain parcellation from fMRI
 ==========================================================
 
-We use spatially-constrained Ward-clustering, KMeans, and Recursive Neighbor
-Agglomeration (ReNA) to create a set of parcels.
+We use spatially-constrained Ward-clustering, KMeans, Hierarchical KMeans
+and Recursive Neighbor Agglomeration (ReNA) to create a set of parcels.
```

Comment:
Is there a reference we can add for this?

Reply:
To document the class ?? Unfortunately this has not yet been clearly used in a publication. This has been raised in the related issue but people decided to push forward anyhow

Reply:
indeed.

Reply:
You're right indeed. 
Closing this conversation...